index.js

```diff
@@ -1,6 +1,51 @@
-let square = document.getElementById("square");
+
+const square = document.getElementById("square");
 console.log(square.style.left, square.style.top);
 
-document.addEventListener("keydown", function(e) {
-  console.log(e.key)
+const keyIndicators = ['ArrowDown', 'ArrowLeft', 'ArrowRight', 'ArrowUp'];
+
+document.addEventListener("keydown", function down(e) {
+  let parsedDown = parseInt(square.style.top)
+  if (e.key === keyIndicators[0] && parsedDown < 300){
+    square.style.top = parsedDown + 10 + 'px'
+  }
+  if (parsedDown > 300){
+    document.removeEventListener("keydown", down);
+  }
 });
+
+document.addEventListener("keydown", function left(e) {
+  let parsedLeft = parseInt(square.style.left)
+  if (e.key === keyIndicators[1] && parsedLeft > 0){
+    square.style.left = parsedLeft - 10 + 'px';
+  }
+  else if (parsedLeft < 0){
+    document.removeEventListener("keydown", left);
+  }
```

Comment:
This code is unreachable since you've already coded the square to stop once it hits a wall. As a best practice, you can remove it.